src/eiffel_graphql_api/graphql/schemas/events/json_schemas/EiffelActivityStartedEvent.json

```diff
@@ -52,24 +56,69 @@
         "security": {
           "type": "object",
           "properties": {
-            "sdm": {
+            "authorIdentity": {
+              "type": "string"
+            },
+            "integrityProtection": {
               "type": "object",
               "properties": {
-                "authorIdentity": {
+                "signature": {
                   "type": "string"
                 },
-                "encryptedDigest": {
+                "alg": {
+                  "type": "string",
+                  "enum": [
+                    "HS256",
+                    "HS384",
+                    "HS512",
+                    "RS256",
+                    "RS384",
+                    "RS512",
+                    "ES256",
+                    "ES384",
+                    "ES512",
+                    "PS256",
+                    "PS384",
+                    "PS512"
+                  ]
+                },
+                "publicKey": {
                   "type": "string"
                 }
               },
               "required": [
-                "authorIdentity",
-                "encryptedDigest"
+                "signature",
+                "alg"
               ],
               "additionalProperties": false
+            },
+            "sequenceProtection": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "sequenceName": {
+                    "type": "string"
+                  },
+                  "position": {
+                    "type": "integer"
+                  }
+                },
+                "additionalProperties": false,
+                "required": [
+                  "sequenceName",
+                  "position"
+                ]
+              }
```

Comment:
This diff is really weird. It looks like the ActS and ActT schemas currently on master are very out of date.

Reply:
They are, yes :) Been bad at updating